test/cli.js

```diff
@@ -21,7 +21,7 @@ test('Exits w/ message when param required by "Web Store Upload" is not provided
 
 test('It should not exit when --source param is not provided', async t => {
     try {
-        await execa('../index.js', ['upload'], {
+        await execa('./index.js', ['upload'], {
```

Comment:
Why was this changed? It should not load the file in the test folder but the one at the root

Reply:
All the file paths are resolved from the root now. Leaving them as-is makes `ava` complain about the fixtures directory not existing.